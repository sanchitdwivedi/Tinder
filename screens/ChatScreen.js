import { View, Text } from "react-native";
import React from "react";
import { SafeAreaView } from "react-native-safe-area-context";
import Header from "../components/Header";
import ChatList from "../components/ChatList";
import { logger } from 'react-native-logs';
 
var log = logger.createLogger();

log.info('Chat screen opened');

const ChatScreen = () => {
  return (
    <SafeAreaView>
      <Header title="Chat" />
      <ChatList />
    </SafeAreaView>
  );
};

export default ChatScreen;
