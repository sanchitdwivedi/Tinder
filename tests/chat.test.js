import React from 'react';
import renderer from 'react-test-renderer'
import ChatScreen from '../screens/ChatScreen'

jest.mock('@react-navigation/core');

describe('Login', () => {
    describe('Rendering', () => {
        it("renders login screen without crash", () => {
            const chat = renderer.create(<ChatScreen/>).toJSON();
            expect(chat).toMatchSnapshot();
        })
    });
});