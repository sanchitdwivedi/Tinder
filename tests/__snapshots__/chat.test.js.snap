// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Rendering renders login screen without crash 1`] = `
<RNCSafeAreaView>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 8,
        "paddingLeft": 8,
        "paddingRight": 8,
        "paddingTop": 8,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
            "paddingBottom": 8,
            "paddingLeft": 8,
            "paddingRight": 8,
            "paddingTop": 8,
          }
        }
      >
        <Text />
      </View>
      <Text
        style={
          Object {
            "fontSize": 24,
            "fontWeight": "bold",
            "lineHeight": 32,
            "paddingLeft": 8,
          }
        }
      >
        Chat
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingBottom": 20,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 18,
          "lineHeight": 28,
          "textAlign": "center",
        }
      }
    >
      No matches at the moment ☹️
    </Text>
  </View>
</RNCSafeAreaView>
`;
